maquetado/scss/styles.scss

```diff
@@ -8,6 +8,50 @@
   max-width: 1000px;
   padding: 20px;
   width: 100%;
+}
+
+.books-list-container {
+  padding: 60px;
+}
+
+.books-list {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+}
+
+.book-preview {
+  margin: 10px;
+  padding: 30px 30px;
+  background-color: $white;
+  box-shadow: 5px 10px $black-05;
```

Comment:
Try `5px 6px 8px rgba(0, 0, 0, 0.4);`, it looks better. This lacks blur. Remind me to show you a trick with box-shadow later.